docs/UserGuide.md

```diff
@@ -81,10 +81,11 @@ LE TRACKER is a gamified tracking application that allows fast typist to easily
 
 ### Edit
 
-- `edit {module_code} [/code {updated_code}] [/name {updated_name}]`: Edits the details of a module in Le Tracker
-- `edit {lecture_name} [/mod {module_code}] [/name {updated_name}]`: Edits the details of a lecture
-- `edit {video_name} [/mod {module_code}] [/lec {lecture_name}] [/name {updated_name}]`: Edits the details of a video
-- Mark/Unmark
+- `edit {module_code} [/code {updated_code}] [/name {updated_name}] [/tags {tag_1}, [{tag_2}, ...]]`: Edits the details of a module in Le Tracker
```

Comment:
Should the arg be [/mod {module_code}] ... instead of /code ...?

Reply:
It is "/code" for specifying updated module code. "/mod" is reserved for specifying context.

Reply:
Sure that sounds logical